QUEST_LV_0100.tsv

```diff
@@ -8122,7 +8122,7 @@ QUEST_LV_0100_20150918_008121	It's been too long, my men should've come back, bu
 QUEST_LV_0100_20150918_008122	
 QUEST_LV_0100_20150918_008123	
 QUEST_LV_0100_20150918_008124	How is it? There are a lot of useful skills to use.{nl}Using skills in an effective manner is very useful in combat.{nl}
-QUEST_LV_0100_20150918_008125	
+QUEST_LV_0100_20150918_008125	If that's the case you should feed yourself to and get rid of the Mausoleum's protector, Doyorushran. I will only say this once, so you better listen well.
```

Comment:
I can't come up with the right words at the moment, but the first part mentions the Royal Mausoleum Guardian at Sesija Entrance.